src/acl.c

```diff
@@ -447,19 +486,45 @@ int ACLCountCategoryBitsForUser(user *u, unsigned long *on, unsigned long *off,
     if (!cflag) return C_ERR;
 
     *on = *off = 0;
-    dictIterator *di = dictGetIterator(server.orig_commands);
+    ACLCountCategoryBitsForUserLogic(server.orig_commands, u, on, off, cflag);
+    return C_OK;
+}
+
+sds ACLDescribeUserCommandRulesSingleCommands(user *u, user *fakeuser, sds rules, dict *commands) {
+    /* Fix the final ACLs with single commands differences. */
```

Comment:
I think that comment is misplaced.. Already included (kept) in the caller where it makes more sense. 
Am I right?

Reply:
yes, it was copied by mistake (appeared twice in the code)